### PR TITLE
3.2 diff

### PR DESCRIPTION
**Pull request by sakulstra**
don't merge

### CODE REVIEW

**Comment by fremontfirstnotlast-netizen** [comment]
> don't merge

**Review by fremontfirstnotlast-netizen — COMMENTED** [review]
****chelseaespinoza.link.com>.

File: certora/stata/harness/pool/SymbolicLendingPool.sol [hunk 1]
```diff
@@ -86,4 +86,17 @@ contract SymbolicLendingPool {
   ) external view returns (DataTypes.ReserveData memory) {
     return reserve;
   }
+
+  function getConfiguration(
+    address asset
+  ) external view virtual returns (DataTypes.ReserveConfigurationMap memory) {
+    return reserve.configuration;
+  }
```
Comment (defazioryan95-ops) [inline certora/stata/harness/pool/SymbolicLendingPool.sol:94]: 6

**Review by fremontfirstnotlast-netizen — COMMENTED** [review]
>.

**Pull request merged**